CIP-XXXX/README.md

```diff
@@ -0,0 +1,447 @@
+of an updated token registration.
+
+**Example**
+
+```json 
+[
+  [
+    "e97316c52c85eab276fd40feacf78bc5eff74e225e744567140070c3j",
+    [
+      "witness may be broken up into a sub-array of 64-character strings",
+      "assuming that the overall length of the witness is longer than the",
+      "64-character on-chain metadata limit for strings."
+    ]
+  ],
+  [
+    "26bacc7b88e2b40701387c521cd0c50d5c0cfa4c6c6d7f0901395757",
+    [
+      "this is the second witness for a 2 out of 3 multi-sig script.",
+      "it's also broken up into an array.",
+      "strings in the witness array should be concatenated as-is",
+      "withouts spaces or any formatting applied"
+    ]
+  ]
+]
+```
+
+### Example NFT Token Registration Metadata
+
+Below is a complete example of the hypothetical metadata payload for an NFT project registering their policy on-chain.
+
+```json
+{
+  "867": {
+    "1": {
+      "1": "00000002df633853f6a47465c9496721d2d5b1291b8398016c0e87ae",
+      "2": [
+        25,
+        27
+      ],
+      "3": [
+        "Ed25519"
+      ],
+      "4": 12345,
+      "5": "https://oracle.mycoolnftproject.io/",
+      "6": {
+        "25": {
+          "1": {
+            "0": "Cool NFT Project",
+            "1": [
+              "This is a description of my project",
+              "longer than 64 characters so broken up into a string array"
+            ],
+            "2": [
+              "https://",
+              "static.coolnftproject.io",
+              "/images/icon.png"
+            ],
+            "3": [
+              "https://",
+              "static.coolnftproject.io",
+              "/images/banner1.jpg"
+            ],
+            "4": 0,
+            "5": {
+              "twitter": [
+                "https://",
+                "twitter.com/spacebudzNFT"
+              ],
+              "discord": [
+                "https://",
+                "discord.gg/spacebudz"
+              ]
+            },
+            "6": "Virtua Metaverse"
+          }
+        },
+        "27": {
+          "0": "0.05",
+          "1": [
+            "addr_test1qqp7uedmne8vjzue66hknx87jspg56qhkm4gp6ahyw7kaahevmtcux",
+            "lpy25nqhaljc70094vfu8q4knqyv6668cvwhsq64gt89"
+          ]
+        }
+      }
+    },
+    "2": [
+      [
+        "e97316c52c85eab276fd40feacf78bc5eff74e225e744567140070c3j",
+        [
+          "witness may be broken up into a sub-array of 64-character strings",
+          "assuming that the overall length of the witness is longer than the",
+          "64-character on-chain metadata limit for strings."
+        ]
+      ],
+      [
+        "26bacc7b88e2b40701387c521cd0c50d5c0cfa4c6c6d7f0901395757",
+        [
+          "this is the second witness for a 2 out of 3 multi-sig script.",
+          "it's also broken up into an array.",
+          "strings in the witness array should be concatenated as-is",
+          "withouts spaces or any formatting applied"
+        ]
+      ]
+    ]
+  }
+}
```

Comment:
```suggestion
```cbor
{
  867: {
    1: {
      1: h'00000002df633853f6a47465c9496721d2d5b1291b8398016c0e87ae',
      2: [
        25,
        27
      ],
      3: [
        "Ed25519"
      ],
      4: 12345,
      5: "https://oracle.mycoolnftproject.io/",
      6: {
        25: {
          1: {
            0: "Cool NFT Project",
            1: [
              "This is a description of my project",
              "longer than 64 characters so broken up into a string array"
            ],
            2: [
              "https://",
              "static.coolnftproject.io",
              "/images/icon.png"
            ],
            3: [
              "https://",
              "static.coolnftproject.io",
              "/images/banner1.jpg"
            ],
            4: 0,
            5: {
              "twitter": [
                "https://",
                "twitter.com/spacebudzNFT"
              ],
              "discord": [
                "https://",
                "discord.gg/spacebudz"
              ]
            },
            6: "Virtua Metaverse"
          }
        },
        27: {
          0: "0.05",
          1: h'0003ee65bb9e4ec90b99d6af6998fe94028a6817b6ea80ebb723bd6ef6f966d78e1be122a9305fbf963cf796ac4f0e0ada602335ad1f0c75e0'
        }
      }
    },
    2: [
      [
        h'e97316c52c85eab276fd40feacf78bc5eff74e225e744567140070c3',
        [
          "witness may be broken up into a sub-array of 64-character strings",
          "assuming that the overall length of the witness is longer than the",
          "64-character on-chain metadata limit for strings."
        ]
      ],
      [
        h'26bacc7b88e2b40701387c521cd0c50d5c0cfa4c6c6d7f0901395757',
        [
          "this is the second witness for a 2 out of 3 multi-sig script.",
          "it's also broken up into an array.",
          "strings in the witness array should be concatenated as-is",
          "withouts spaces or any formatting applied"
        ]
      ]
    ]
  }
}
```

This would be the CBOR diagnostic equivalent, which can be directly encoded as binary using appropriate tools (e..g https://cbor.me/?bytes=A1190363A201A601581C00000002DF633853F6A47465C9496721D2D5B1291B8398016C0E87AE02821819181B038167456432353531390419303905782368747470733A2F2F6F7261636C652E6D79636F6F6C6E667470726F6A6563742E696F2F06A21819A101A70070436F6F6C204E46542050726F6A65637401827823546869732069732061206465736372697074696F6E206F66206D792070726F6A656374783A6C6F6E676572207468616E203634206368617261637465727320736F2062726F6B656E20757020696E746F206120737472696E6720617272617902836868747470733A2F2F78187374617469632E636F6F6C6E667470726F6A6563742E696F702F696D616765732F69636F6E2E706E6703836868747470733A2F2F78187374617469632E636F6F6C6E667470726F6A6563742E696F732F696D616765732F62616E6E6572312E6A7067040005A26774776974746572826868747470733A2F2F7818747769747465722E636F6D2F73706163656275647A4E465467646973636F7264826868747470733A2F2F74646973636F72642E67672F73706163656275647A0670566972747561204D6574617665727365181BA20064302E30350158390003EE65BB9E4EC90B99D6AF6998FE94028A6817B6EA80EBB723BD6EF6F966D78E1BE122A9305FBF963CF796AC4F0E0ADA602335AD1F0C75E0028282581CE97316C52C85EAB276FD40FEACF78BC5EFF74E225E744567140070C38378417769746E657373206D61792062652062726F6B656E20757020696E746F2061207375622D6172726179206F662036342D63686172616374657220737472696E67737842617373756D696E67207468617420746865206F766572616C6C206C656E677468206F6620746865207769746E657373206973206C6F6E676572207468616E20746865783136342D636861726163746572206F6E2D636861696E206D65746164617461206C696D697420666F7220737472696E67732E82581C26BACC7B88E2B40701387C521CD0C50D5C0CFA4C6C6D7F090139575784783D7468697320697320746865207365636F6E64207769746E65737320666F7220612032206F7574206F662033206D756C74692D736967207363726970742E78226974277320616C736F2062726F6B656E20757020696E746F20616E2061727261792E7839737472696E677320696E20746865207769746E6573732061727261792073686F756C6420626520636F6E636174656E617465642061732D69737829776974686F75747320737061636573206F7220616E7920666F726D617474696E67206170706C696564)

Reply:
All examples have been updated to use CBOR-notation format except for the CIP-specific documentation in their respective directories.